tools/src/main/python/projadm.py

```diff
@@ -281,6 +292,7 @@ def project_delete(doit, logger, project, uri):
                                basedir=args.base,
                                uri=uri,
                                configmerge=configmerge_file,
```

Comment:
what if the configmerge_file is not defined here? can it happen?

Reply:
If `roconfig` is not set, then it does not matter if `configmerge` is set. Otherwise, it will be checked in the block above.